API/Models/Errors/Auth/AuthErrorCode.cs

```diff
@@ -19,4 +19,9 @@ public enum AuthErrorCode
     /// The token has expired and is no longer valid.
     /// </summary>
     EXPIRED_TOKEN,
+
+    /// <summary>
+    /// The user is not authorized to access the requested resource.
+    /// </summary>
+    INVALID_PASSWORD,
```

Comment:
W issue wspomniałem o `INVALID_PASSWORD`, jednak dopisanie CURRENT da więcej kontekstu.
```suggestion
    /// <summary>
    /// The provided current password is incorrect.
    /// </summary>
    /// <remarks>
    /// Used when the user attempts to change their password and the provided current password
    /// does not match the stored password.
    /// </remarks>
    INVALID_CURRENT_PASSWORD,
```